scripts/cnv_wdl/germline/cnv_germline_panel_creation_workflow.wdl

```diff
@@ -63,8 +64,10 @@ workflow CNVGermlinePanelWorkflow {
   if (!is_wgs) {
     call CNVTasks.PadTargets {
       input:
-        targets = targets,
-        gatk_jar = gatk_jar
+        # This is a bit of a hack.  The task will fail if targets is not defined when it gets here.
```

Comment:
Are this comment and the `select_first` necessary?  This task is inside a block that ensures `targets` will be defined.